cfgov/unprocessed/js/modules/util/dom-traverse.js

```diff
@@ -1,7 +1,6 @@
 'use strict';
 
-// Required polyfills for IE9.
-if ( window.Modernizr && !window.Modernizr.classlist ) { require( '../polyfill/class-list' ); } // eslint-disable-line no-undef, global-require, no-inline-comments, max-len
+require( '../polyfill/class-list' );
```

Comment:
Didn't we add `if ( window.Modernizr && !window.Modernizr.classlist )` so class-list polyfill isn't delivered to all browsers?